central/graphql/resolvers/image_vulnerabilities_test.go

```diff
@@ -52,14 +52,17 @@ func (s *GraphQLImageVulnerabilityTestSuite) SetupSuite() {
 	s.ctx = loaders.WithLoaderContext(sac.WithAllAccess(context.Background()))
 	mockCtrl := gomock.NewController(s.T())
 	s.testDB = SetupTestPostgresConn(s.T())
+	vulnReqDatastore, err := TestVulnReqDatastore(s.T(), s.testDB)
```

Comment:
I venture to guess that at the end of the day, none of these stores can actually return an err any longer.  I'm going to take a look at that when I have a minute and fix it.  Just making you aware of that.